### PR TITLE
Fix k6 browser TD for waitForFunction options

### PR DESCRIPTION
**Pull request by ankur22**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Review by olegbespalov — APPROVED** [review]

File: types/k6/browser/index.d.ts [hunk 1]
```diff
@@ -393,19 +393,16 @@ export interface ScreenshotOptions {
  * - `mutation` - use a mutation observer
  * - `interval` - use a polling interval
  */
-export type PollingMethod = "raf" | "mutation" | "interval";
+export type PollingMethod = number | "raf" | "mutation";
```
Comment (joanlopez) [inline types/k6/browser/index.d.ts:396]: Out of curiosity; what is `"mutation"` used for? I couldn't find it in our docs, nor in Playwright docs 🤔

Comment (ankur22) [reply types/k6/browser/index.d.ts:396]: It works with [mutationObserver](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver) for changes in the DOM and calls the observer when anything changes.

Comment (ankur22) [reply types/k6/browser/index.d.ts:396]: I've updated it in the TD: 76e4a3ddfa8d02eecfb6794272f652269d61a1b9

Comment (joanlopez) [reply types/k6/browser/index.d.ts:396]: Thanks! 🙌🏻

**Review by joanlopez — APPROVED** [review]

**Comment by ankur22** [comment]
Ready to merge

**Pull request merged**